gems/faraday/2.5/manifest.yaml

```diff
@@ -0,0 +1,7 @@
+# manifest.yaml describes dependencies which do not appear in the gemspec.
+# If this gem includes such dependencies, comment-out the following lines and
+# declare the dependencies.
+# If all dependencies appear in the gemspec, you can remove this file.
```

Comment:
Could you remove this file?

Reply:
sure. remove on this commit.
https://github.com/ruby/gem_rbs_collection/pull/198/commits/b6f2e9a4da70affb82499645ac8ad35781052547